### PR TITLE
Runtime 7.2 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -24,6 +24,32 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+
+- Be more selective when adopting pods with KubernetesExecutor ([28899](https://github.com/apache/airflow/pull/28899))
+- KubenetesExecutor sends state even when successful ([28871](https://github.com/apache/airflow/pull/28871))
+- Annotate KubeExecutor pods that we don't delete ([28844](https://github.com/apache/airflow/pull/28844))
+
+### Additional improvements
+
+- Upgraded `astro-sdk-python` to 1.4.0, which includes support for FTP/SFTP, Databricks, and Azure Blob Storage. For a complete list of changes, see the [Astro Python SDK Changelog](https://astro-sdk-python.readthedocs.io/en/1.4/CHANGELOG.html).
+- Upgraded `astronomer-providers` to 1.13.0, which includes support for using a role ARN with `AwsBaseHookAsync`. For a complete list of changes, see the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.13.0/CHANGELOG.rst).
+- Upgraded `openlineage-airflow` to 0.19.2, which includes new support for Airflow operators like the `S3FileTransformOperator` and additional facets for task runs. See the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/releases/tag/0.19.2) for more information.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:51]:
```suggestion
- Upgraded `astro-sdk-python` to 1.4.0, which includes support for FTP/SFTP, Databricks, and Azure Blob Storage. See the [Astro Python SDK Changelog](https://astro-sdk-python.readthedocs.io/en/1.4/CHANGELOG.html) for a complete list of changes.
- Upgraded `astronomer-providers` to 1.13.0, which includes support for using a role ARN with `AwsBaseHookAsync`. See the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.13.0/CHANGELOG.rst) for a complete list of changes.
- Upgraded `openlineage-airflow` to 0.19.2, which includes new support for Airflow operators like the `S3FileTransformOperator` and additional facets for task runs. See the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/releases/tag/0.19.2) for a complete list of changes.
```

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -24,6 +24,32 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
 
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Astro Runtime 7.2.0
+
+- Release date: January 20, 2022
```
Comment (kwcanuck) [inline astro/runtime-release-notes.md:29]:
```suggestion
- Release date: January 20, 2023
```

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -24,6 +24,32 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+- Airflow version: 2.5.1
+
+### Airflow 2.5.1
+
+Astro Runtime 7.2.0 includes same-day support for Airflow 2.5.1, which includes a collection of bug fixes. Fixes include:
```
Comment (kwcanuck) [inline astro/runtime-release-notes.md:34]:
```suggestion
Astro Runtime 7.2.0 includes same-day support for Airflow 2.5.1. Airflow 2.5.1 contains a number of bug fixes including:
```

File: astro/runtime-release-notes.md [hunk 4]
```diff
@@ -24,6 +24,32 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+
+- Return list of tasks that will be queued([28066](https://github.com/apache/airflow/pull/28066))
```
Comment (kwcanuck) [inline astro/runtime-release-notes.md:36]:
```suggestion
- Return list of tasks that will be queued ([28066](https://github.com/apache/airflow/pull/28066))
```

File: astro/runtime-release-notes.md [hunk 5]
```diff
@@ -24,6 +24,32 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+
+### Early access Airflow bug fixes
+
+- Be more selective when adopting pods with KubernetesExecutor ([28899](https://github.com/apache/airflow/pull/28899))
+- KubenetesExecutor sends state even when successful ([28871](https://github.com/apache/airflow/pull/28871))
+- Annotate KubeExecutor pods that we don't delete ([28844](https://github.com/apache/airflow/pull/28844))
+
+### Additional improvements
+
+- Upgraded `astro-sdk-python` to 1.4.0, which includes support for FTP/SFTP, Databricks, and Azure Blob Storage. See the [Astro Python SDK Changelog](https://astro-sdk-python.readthedocs.io/en/1.4/CHANGELOG.html) for a complete list of changes.
```
Comment (jedcunningham) [inline astro/runtime-release-notes.md:49]: This is the first time Astro SDK is installed by default, so this is new not upgraded.

**Pull request merged**